generate/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+	"fmt"
+	"os"
+	"strings"
+
+	mh "github.com/multiformats/go-multihash"
+)
+
+const bits = 16
+const target = 1 << bits
+const idLen = 32 + 2
+
+func main() {
+	pkg := os.Getenv("GOPACKAGE")
+	file := os.Getenv("GOFILE")
+	targetFile := strings.TrimSuffix(file, ".go") + "_prefixmap.go"
+
+	ids := new([target]uint64)
+	found := new([target]bool)
+	count := int32(0)
+
+	out := make([]byte, 32)
+	inp := [idLen]byte{mh.SHA2_256, 32}
+	hasher := sha256.New()
+
+	for i := uint64(0); count < target; i++ {
+		binary.BigEndian.PutUint64(inp[2:], i)
```

Comment:
Now that I think of it, you are right.  This will work just as well if I put the integer in the same position while crafting the multihash.
This just seemed liked a good idea at that time :p